From: =?utf-8?q?R=C3=A9mi_Palancher?= <remi-externe.palancher@edf.fr>
Date: Fri, 3 Jan 2020 09:17:45 +0100
Subject: Search slurm lib in slurm-wlm subdir

Debian packages actually install slurm plugins shared libraries into
slurm-wlm subdirectory. This patch makes setup.py properly looks into
this debian specific subdirectory.
---
 setup.py | 2 +-
 1 file changed, 1 insertions(+), 1 deletions(-)

diff --git a/setup.py b/setup.py
index 8ce8814..ca0d78b 100644
--- a/setup.py
+++ b/setup.py
@@ -117,7 +117,7 @@ class Pyslurm:
     def make_extension(self, extension_name):
         """Generate an Extension object from its dotted name"""
         extension_path = extension_name.replace(".", os.path.sep) + ".pyx"
-        runtime_library_dirs = [self.slurm_lib, f"{self.slurm_lib}/slurm"]
+        runtime_library_dirs = [self.slurm_lib, f"{self.slurm_lib}/slurm-wlm"]
         return Extension(
             extension_name,
             [extension_path],
